### PR TITLE
AES256 password encoding - FAT test updates

### PR DESCRIPTION
**Pull request by Zech-Hein**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

################################################################################################

### CODE REVIEW

**Comment by Zech-Hein** [comment]
#build

File: dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java [hunk 1]
```diff
@@ -130,22 +130,56 @@ public void testUtil() {
...
-                     PasswordUtil.decode("{aes}AEmVKa+jOeA7pos+sSfpHNmH1MVfwg8ZoV29iDi6I0ZGcov6hSZsAxMhFr91jTSBYQ=="));
-        assertEquals("Did not decode password encoded with AES_V1 (AES-256) encoded password", "alternatepwd",
-                     PasswordUtil.decode("{aes}ARCejBipVe2pCLpoNe9iRREPNbQmPVSfXOe5kHZneiYVR+nSXJ8UmFzlNtwrQgKwnrkaDJmNnr+GpDWCOk1MPY7VbvR0/FBUrIENeU5+L9e8a1K4YvWeQ2UUbLv97SHa3P9Iky7rIbeQ9l4Xi0q0"));
+            assertEquals("Did not decode password encoded with AES_V0 (AES-128) encoded password", "alternatepwd",
+                         PasswordUtil.decode("{aes}AEmVKa+jOeA7pos+sSfpHNmH1MVfwg8ZoV29iDi6I0ZGcov6hSZsAxMhFr91jTSBYQ=="));
+
+            // decode("{aes}<valid AES256 encoded password>); should fail with InvalidPasswordDecodingException when beta is disabled.
+            try {
+                PasswordUtil.decode("{aes}ARCejBipVe2pCLpoNe9iRREPNbQmPVSfXOe5kHZneiYVR+nSXJ8UmFzlNtwrQgKwnrkaDJmNnr+GpDWCOk1MPY7VbvR0/FBUrIENeU5+L9e8a1K4YvWeQ2UUbLv97SHa3P9Iky7rIbeQ9l4Xi0q0");
+                fail("decode AES256 encoded password should have failed with InvalidPasswordDecodingException when beta is disabled.");
```
Comment (mshah0722) [inline dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java:176]: fail("Expected InvalidPasswordDecodingException when decoding an AES-256 password with beta disabled.");

Comment (mshah0722) [inline dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java:176]: fail("Expected InvalidPasswordDecodingException when decoding an AES-256 password with beta disabled.");

**Review by mshah0722 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java (hunk 1 above)
Comment (mshah0722) [inline dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java:176]:
```suggestion
                fail("Expected InvalidPasswordDecodingException when decoding an AES-256 password with beta disabled.");
```

**Comment by Zech-Hein** [comment]
#build

**Review by mshah0722 — DISMISSED** [review]
LGTM!

**Comment by Zech-Hein** [comment]
#build

Added jvm.options updates for enabling beta mode

**Review by mshah0722 — APPROVED** [review]
LGTM!

**Pull request merged**